### PR TITLE
Add unified order editing feature flag

### LINKED ISSUE #6582: Add unified order editing feature flag

### PR DESCRIPTION
**Pull request by wzieba**
Closes: #6582

### Description

Every great feature starts with a feature flag.

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt [hunk 1]
```diff
@@ -9,11 +9,11 @@ enum class FeatureFlag {
     DB_DOWNGRADE,
     CARD_READER,
     JETPACK_CP,
-    ORDER_FILTERS,
```
Comment (wzieba) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt:12]: @JorgeMucientes could you please confirm if we can remove this flag? I haven't seen any usage in the project so I thought we might do this.

Comment (JorgeMucientes) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt:12]: Hey @wzieba. Yes! We can remove that. Order filters was enabled for all users a long time ago. Thanks for cleaning this!

Comment (wzieba) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt:12]: Thanks!

**Review by nbradbury — APPROVED** [review]
:shipit:

**Pull request merged**